### DISCUSSION TITLE
ActiveDoor control

### DISCUSSION THREAD

**Discussion by joydashy** [astrandb/miele#537] [General]
The Miele K 7797 C R (e.g.) supports ActiveDoor, with knock2open. It has integration with Miele@Home (for Google Assistant). I wonder if anyone knows or has tried if it would also be possible to open the door through the API? That would allow for some cool integration, such as a wall button on the way to the kitchen to arrive at an open door :)

**Comment by ChristophWeigert** [comment]
I'm also interested. Did you find out?

**Comment by manuelrus** [comment]
I'm also interested - anyone with an solution?
